pandas/conftest.py

```diff
@@ -97,3 +97,22 @@ def join_type(request):
     Fixture for trying all types of join operations
     """
     return request.param
+
+
+TIMEZONES = [None, 'UTC', 'US/Eastern', 'Asia/Tokyo', 'dateutil/US/Pacific']
+
+
+@pytest.fixture(params=TIMEZONES)
+def tz_naive(request):
+    """
+    Fixture for trying timezones including default (None)
```

Comment:
can you make this ore explicit (e.g. list the timezones here); you can programatically construct the doc string I think (to make this DRY)

Reply:
I added decorator for this